packages/node_modules/@webex/plugin-meetings/src/ciscospark.js

```diff
@@ -16,7 +16,7 @@ CiscoSpark.version = PACKAGE_VERSION;
 
 CiscoSpark.init = function init(attrs = {}) {
   const config = merge({}, {
-    hydraServiceUrl: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
+    hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
```

Comment:
Changing these config value names is going to be a breaking change for third party developers if they overwrite or define these urls. @mattnorris do you know the likelihood of that?

@stanjiawang is there any way to make this backward compatible with a deprecation warning?

Reply:
Agreed. `hydraServiceUrl` is used in a few different plugins and probably shouldn't be changed unless there is a real need to.

Reply:
The contrary to this is altering URLs for integration, which we just went through with BTS. However, if we ensure that Hydra's URL is changed depending on another environment variable, it will work. 

Do we know the cases where Hydra will have to be integration?

Reply:
@andrewholsted @adamweeks @mattnorris Thanks a lot for your comments! As I mentioned in the PR description, the key to the service URL map has been changed on the server side. The res.services(which keys are serviceName + "ServiceUrl") is obsoleted and will be deprecated soon in the response message when registering to wdm and we should use res.serviceHostMap.serviceLinks(which keys are just the service name without the redundant "ServiceUrl").

https://sqbu-github.cisco.com/gist/sgaikwad/879d8506fef720ebdf3f52b876190d8e

The obsoleted map below:
![screen shot 2018-11-14 at 11 37 18 am](https://user-images.githubusercontent.com/14055396/48507764-b1c6b080-e801-11e8-95fb-d2c5eb337158.png)

The new map we should use:
![screen shot 2018-11-14 at 11 35 57 am](https://user-images.githubusercontent.com/14055396/48507692-7b893100-e801-11e8-9091-6cbf6f15c0d2.png)